website/data/alert-banner.js

```diff
@@ -2,12 +2,12 @@ export const ALERT_BANNER_ACTIVE = true
 
 // https://github.com/hashicorp/web-components/tree/master/packages/alert-banner
 export default {
-  linkText: 'Register now',
-  url: 'https://www.hashicorp.com/events/webinars/introducing-nomad-1-0',
+  linkText: 'Learn more!',
+  url: 'https://www.hashicorp.com/blog/announcing-general-availability-of-hashicorp-nomad-1-0',
   tag: 'ANNOUNCEMENT',
   text:
-    'Join us on Oct 27th for the Nomad 1.0 Product Announcement with Armon Dadgar',
+    'Nomad 1.0.0 is now generally available, which includes 5 major new features and many improvements.',
   // Set the `expirationDate prop with a datetime string (e.g. `2020-01-31T12:00:00-07:00`)
   // if you'd like the component to stop showing at or after a certain date
-  expirationDate: '2020-10-27T09:00:00-07:00',
+  expirationDate: '2021-02-08T09:00:00-07:00',
```

Comment:
That seems like an eternity away from now!

Reply:
Ha, right? I think we'll probably have something else to announce before then, so this was chosen to be a just-in-case-we-forget-about-it.